storage/lib/Android.bp

```diff
@@ -40,7 +40,7 @@ android_library {
         enabled: false,
     },
     kotlincflags: [
-        "-Xopt-in=kotlin.RequiresOptIn",
+        "-opt-in=kotlin.RequiresOptIn",
```

Comment:
Is this how they do it now? Is that documented somewhere?

Reply:
This was hinted at in a compile-time warning. As I see it is mentioned here in docs: https://kotlinlang.org/docs/opt-in-requirements.html#module-wide-opt-in